awx/main/analytics/collectors.py

```diff
@@ -69,6 +69,54 @@ def four_hour_slicing(key, since, until, last_gather):
         start = end
 
 
+def host_metric_slicing(key, since, until, last_gather, **kwargs):
+    """
+    Slicing doesn't start 4 weeks ago, but sends whole table monthly or first time
+    """
+    from awx.main.models.inventory import HostMetric
+
+    if since is not None:
```

Comment:
isn't `since` always passed here?

Reply:
`since` is `None ` by default: 
- https://github.com/ansible/awx/blob/devel/awx/main/analytics/core.py#L164
- the only way how it isn't None is CLI `awx-manage gather_analytics --since ...`